endpointmanager/pkg/endpointmanager/postgresql/healthitproductstore.go

```diff
@@ -2,11 +2,19 @@ package postgresql
 
 import (
 	"context"
+	"database/sql"
 	"encoding/json"
 
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager"
 )
 
+// prepared statements are left open to be used throughout the execution of the application
+// TODO: figure out if there's a better way to manage this for bulk calls
+var areHealthITProductStatementsPrepared = false
```

Comment:
I think that if you make prepareHealthITProductStatements return the prepared statements then you can just assign the gobal variables their values here and then won't need to use  the areHealthITProductStatementsPrepared function since the prepare statement variables will have their values assigned at package initialization.